frostsnapp/native/src/api.rs

```diff
@@ -264,9 +291,11 @@ pub fn turn_stderr_logging_on(level: LogLevel) {
         })
         .without_time()
         .pretty()
-        .finish();
+        .finish()
+        .with(StreamLogsToDart);
+
     let _ = tracing::subscriber::set_global_default(subscriber);
-    event!(Level::INFO, "logging to stderr");
+    event!(Level::INFO, "logging to stderr and Dart logger");
 }
 
 pub fn turn_logcat_logging_on(_level: LogLevel) {
```

Comment:
I think this should return the Stream of log rather than there being a separate API for this.